qiskit-superstaq/qiskit_superstaq/daily_integration_test.py

```diff
@@ -234,6 +234,22 @@ def test_dfe(provider: qss.superstaq_provider.SuperstaqProvider) -> None:
     assert isinstance(result, float)
 
 
+def test_aces(provider: qss.superstaq_provider.SuperstaqProvider) -> None:
+    backend = provider.get_backend("ss_unconstrained_simulator")
+    job_id = backend.submit_aces(
+        qubits=qiskit.QuantumRegister(1),
```

Comment:
This throws `FAILED qiskit-superstaq/qiskit_superstaq/daily_integration_test.py::test_aces - TypeError: Object of type QuantumRegister is not JSON serializable` when i run it on the local server. Why is it not 
```suggestion
        qubits=[0],
```

Like the `test_aces` in `cirq-superstaq?

Reply:
Yes this should be `qubits=[0]`, I didn't update this test, sorry about that. I changed the `qubits` argument from accepting a `qiskit.QuantumRegister` to a `Sequence[int]` since it is harder to specify non-linear qubits with `qiskit.QuantumRegister`.